web/core/components/issues/issue-layouts/list/blocks-list.tsx

```diff
@@ -40,26 +41,30 @@ export const IssueBlocksList: FC<Props> = (props) => {
     <div className="relative h-full w-full">
       {issueIds &&
         issueIds.length > 0 &&
-        issueIds.map((issueId: string, index: number) => (
-          <IssueBlockRoot
-            key={issueId}
-            issueIds={issueIds}
-            issueId={issueId}
-            issuesMap={issuesMap}
-            updateIssue={updateIssue}
-            quickActions={quickActions}
-            canEditProperties={canEditProperties}
-            displayProperties={displayProperties}
-            nestingLevel={0}
-            spacingLeft={0}
-            containerRef={containerRef}
-            selectionHelpers={selectionHelpers}
-            groupId={groupId}
-            isLastChild={index === issueIds.length - 1}
-            isDragAllowed={isDragAllowed}
-            canDropOverIssue={canDropOverIssue}
-          />
-        ))}
+        issueIds.map(
+          (issueId: string, index: number) =>
+            issuesMap[issueId].created_at && (
+              <IssueBlockRoot
+                key={issueId}
+                issueIds={issueIds}
+                issueId={issueId}
+                issuesMap={issuesMap}
+                updateIssue={updateIssue}
+                quickActions={quickActions}
+                canEditProperties={canEditProperties}
+                displayProperties={displayProperties}
+                nestingLevel={0}
+                spacingLeft={0}
+                containerRef={containerRef}
+                selectionHelpers={selectionHelpers}
+                groupId={groupId}
+                isLastChild={index === issueIds.length - 1}
+                isDragAllowed={isDragAllowed}
+                canDropOverIssue={canDropOverIssue}
+                shouldRenderByDefault={isIssueNew(issuesMap[issueId])}
```

Comment:
Here as well, can you just make sure to use the logic from within the `IssueBlockRoot` instead.